Editor/Credentials/Strings.cs

```diff
@@ -105,20 +99,6 @@ public static class Strings
         public const string LabelStackUpdateConsoleWarning = "LabelStackUpdateConsoleWarning";
         public const string LabelStackUpdateRemovalHeader = "LabelStackUpdateRemovalHeader";
         public const string LabelStackUpdateQuestion = "LabelStackUpdateQuestion";
-        public const string StatusLocalTestErrorTemplate = "StatusLocalTestErrorTemplate";
-        public const string StatusLocalTestServerErrorTemplate = "StatusLocalTestServerErrorTemplate";
-        public const string StatusLocalTestRunning = "StatusLocalTestRunning";
-        public const string TitleLocalTesting = "TitleLocalTesting";
-        public const string LabelLocalTestingHelp = "LabelLocalTestingHelp";
-        public const string LabelLocalTestingWindowsServerPath = "LabelLocalTestingWindowsServerPath";
-        public const string LabelLocalTestingMacOsServerPath = "LabelLocalTestingMacOsServerPath";
-        public const string TitleLocalTestingServerPathDialog = "TitleLocalTestingServerPathDialog";
-        public const string TooltipLocalTestingServerPath = "TooltipLocalTestingServerPath";
-        public const string LabelLocalTestingJarPath = "LabelLocalTestingJarPath";
-        public const string LabelLocalTestingPort = "LabelLocalTestingPort";
-        public const string TooltipLocalTestingPort = "TooltipLocalTestingPort";
-        public const string LabelLocalTestingStartButton = "LabelLocalTestingStartButton";
-        public const string LabelLocalTestingStopButton = "LabelLocalTestingStopButton";
```

Comment:
Are these just no longer used?

Reply:
They are only used in the Local Testing UI, which is no longer needed.